ballerina-command/src/main/java/org/ballerinalang/command/cmd/UseCommand.java

```diff
@@ -44,7 +44,7 @@ public UseCommand(PrintStream printStream) {
 
     public void execute() {
         if (helpFlag) {
-            printUsageInfo(BallerinaCliCommands.USE);
+            printUsageInfo("dist-" + BallerinaCliCommands.USE);
```

Comment:
We could use a constant for this string ?

Reply:
+1